### ISSUE TITLE
 Pokemon level as filter, or having individual filter

### ISSUE THREAD

**Issue by zno3** [NecronomiconCoding/NecroBot#676]
Because some unevolved pokemon have low cp like 200cp magicarp or 700cp dratini for example, if i make filter out of those 200cp or 700cp, pokemon inventory would be full, those pokemon gain massive amount of cp only if evolved. So having cp as a filter only good for last evolution stage pokemon which mostly have huge amount of cp. But having a massive pokemon filter that override the global settings like this below, also good.
```
#name #mincp #miniv #transfer #skip #evolvecp #evolveiv 
pikachu 800 90 true false 900 95
ratata global global true true 1 1
dratini 700 70 false false 700 90 
magicarp 200 global true false 200 70
```
thats just some example

**Comment by ShawnHardwick** [comment]
I don't think the devs would go for this. You are essentially asking for pokemon specific settings which can already be done with the current lists. If you don't want a pokemon to be transferred, add it to the PokemonsNotToTransfer list.

**Comment by zno3** [comment]
Yeah i think they wont, or maybe something like adding these to filter on not to catch, evolve list or transfer list.
pokemontoevolve:
```
//pokemon, min cp to evo, min iv to evo
magicarp, 200, 90
ratata, 200, 90
```
